sdk/src/main/java/com/meniga/sdk/models/challenges/enums/CustomChallengeColor.java

```diff
@@ -20,6 +20,10 @@
 	@SerializedName("danger")
 	RED;
 
+	/**
+	 * Use Colors should be defined in app instead.
```

Comment:
This comment can be prepended with javadoc's `@deprecated Define colours in the app instead.`.

Reply:
Added @deprecated